packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -157,7 +157,9 @@ const es2022 = {
     "Ergonomic brand checks for private fields",
   "proposal-class-properties": {
     features: [
-      "static class fields",
+      "static class fields / public static class fields",
+      "static class fields / private static class fields",
+      "static class fields / computed static class fields",
       "instance class fields, public instance class fields",
```

Comment:
I will open another PR changing `, ` to ` / ` here. I'd prefer to keep it separate because it might cause changes to our compat data and fixtures not related to rhino.